drupal-org.make

```diff
@@ -124,11 +124,6 @@ projects[facetapi][patch][2327997_3] = https://www.drupal.org/files/issues/facet
 ; https://www.drupal.org/node/2811151
 projects[felix][patch][2811151] = https://www.drupal.org/files/issues/felix-invalid-foreach-2811151-1.patch
 
-; Webform
```

Comment:
Accidently commited this .make but thought I reverted it. But yeah it should be fixed in webform encrypt 1.2 but that versionnumber needs to be changed as well.